TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Mqs/EditMq/Specialism/Index.cshtml.cs

```diff
@@ -61,9 +62,10 @@ public override async Task OnPageHandlerExecutionAsync(PageHandlerExecutingConte
         PersonId = personInfo.PersonId;
         PersonName = personInfo.Name;
 
-        Specialisms = MandatoryQualificationSpecialismRegistry.GetAll()
-            .OrderBy(t => t.Title)
-            .ToArray();
+        var migratedFromDqtWithLegacySpecialism = qualificationInfo.MandatoryQualification.DqtSpecialismId is Guid dqtSpecialismId &&
+            MandatoryQualificationSpecialismRegistry.GetByDqtValue((await referenceDataCache.GetMqSpecialismById(dqtSpecialismId)).dfeta_Value).IsLegacy();
+
+        Specialisms = MandatoryQualificationSpecialismRegistry.GetAll(includeLegacy: migratedFromDqtWithLegacySpecialism);
```

Comment:
You would still see other legacy specialisms in this list which could then be potentially selected.
If that's Ok then fine but another option is to just add this single legacy option to the list?

Reply:
I think that's ok here - I wanted to keep it as simple as possible.